.github/workflows/ci.yml

```diff
@@ -26,7 +29,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: olafurpg/setup-scala@v13
+      - uses: actions/setup-java@v3
+        with:
+          distribution: adopt
```

Comment:
```suggestion
          distribution: temurin
```
adopt is no longer maintained and moved to temurin

Reply:
Didn't wanted to change the distribution, but totally agree `temurin` is the way to go

Reply:
I think it's fine to change as you are already addressing tech debt here.

Reply:
perfect, will do so